### DISCUSSION TITLE
Setting keep_alive parameter for Ollama to unload models from vram memory?

### DISCUSSION THREAD

**Discussion by Jonseed** [heshengtao/comfyui_LLM_party#115] [Q&A]
How do you set the `keep_alive` parameter for Ollama API? The default is 5, which keeps models loaded into vram memory for 5 minutes. If the user can set this to 0, then the models are immediately unloaded from vram memory after a request, freeing up vram for other tasks (like image generation).

**Comment by heshengtao** [comment]
The Clear Model node allows you to exit the video memory of the ollama model at any location. You need to insert the Clear Model node between any connections and set the is_ollama to true
